docs/news.rst

```diff
@@ -0,0 +1,5 @@
+Release Notes
+=============
+
+**0.1**
+* Initial release
```

Comment:
These two lines appear in the output doc as `0.1 * Initial release`, where `0.1` is in bold. Is that what you intended? I think the asterisk after the `0.1` looks odd, as if it's pointing at a footnote.